spec/models/optimized_image_spec.rb

```diff
@@ -118,7 +118,7 @@ def local(url)
         end
 
         it "downloads a copy of the original image" do
-          Tempfile.any_instance.expects(:close!)
+          Tempfile.any_instance.expects(:close!).twice
```

Comment:
Just curious, is this related to the problem in this PR? :smile_cat:

Reply:
Yeah that's weird.

Reply:
Tests don't pass locally without it. I'll remove it and run it through again. It must be my environment